src/controllers/admin/UserAjax.php

```diff
@@ -460,5 +460,49 @@ private function getRoleInfo($user)
         );
     }
 
+    public function postChangePass()
+    {
+        $response = array(
+            "method" => "changepass",
+            "success" => false,
+            "error_code" => 0,
+            "error_message" => ""
+        );
 
+        try {
+            $data = json_decode(file_get_contents("php://input"));
+            if (empty($data)) {
+                $data = (object) $_POST;
+            }
+            // Check required fields
+            $requiredParams = array('email', 'newpass');
+            $params = (array) $data;
+            foreach ($requiredParams as $param) {
+                if (empty($params[$param])) {
+                    throw new \Exception(ucfirst($param) .' is required.');
+                }
+            }
+
+            if ($data->currentpass == $data->newpass) {
```

Comment:
Does the admin really know the current password? Most likely not. 

Let's assume if the admin user is making this change, we want to set it to the 'newpass'.